invenio_rdm_records/theme/templates/invenio_rdm_records/macros/detail.html

```diff
@@ -18,6 +18,19 @@
 </div>
 {%- endmacro %}
 
+{% macro show_safe_detail(title, value) %}
+<div class="ui relaxed grid">
+  <div class="two column row">
+    <div class="four wide column">
+      <p><b>{{ title }}</b></p>
+    </div>
+    <div class="twelve wide column">
+      {{ value | safe }}
```

Comment:
Should use [sanitize_html](https://invenio-formatter.readthedocs.io/en/latest/api.html#invenio_formatter.filters.html.sanitize_html) instead